<launch>
    <arg name="motor_left_topic" default="motor_left/speed" />
    <arg name="motor_right_topic" default="motor_right/speed" />
    <arg name="steering_topic" default="steering/angle" />
    <arg name="status_topic" default="drive/status" />
  
    <arg name="name" default="AckermannMonitoringSkill" />
    <arg name="uid" default="ackermannmonitoringskill" />
  
    <node name="ackermann_monitoring_skill" pkg="picarx_ackermann_drive" type="ackermann_monitor_skill.py" respawn="false" output="screen" args="$(arg name) $(arg uid)">
      <param name="motor_left_topic" type="str" value="$(arg motor_left_topic)" />
      <param name="motor_right_topic" type="str" value="$(arg motor_right_topic)" />
      <param name="steering_topic" type="str" value="$(arg steering_topic)" />
      <param name="status_topic" type="str" value="$(arg status_topic)" />
    </node>
  
  </launch>